nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/kudu/PutKudu.java

```diff
@@ -131,8 +115,9 @@
     protected static final PropertyDescriptor INSERT_OPERATION = new Builder()
```

Comment:
I am not sure the best way for Nifi to handle a change like this, but calling this `INSERT_OPERATION` while also supporting DELETE doesn't make a lot of sense. It should probably be something like `OPERATION_TYPE`.